stable/cert-manager/values.yaml

```diff
@@ -69,6 +69,9 @@ ingressShim: {}
   # defaultACMEChallengeType: ""
   # defaultACMEDNS01ChallengeProvider: ""
 
+webhook:
+  enabled: false
```

Comment:
We disable the new webhook chart by default

Reply:
can you add a test to validate when the webhook is true?